DropDownBox: add a Getting Started tutorial (#4159)
DevExpress/devextreme-documentation: concepts/05 UI Components/DropDownBox/00 Getting Started with DropDownBox/20 Handle Events.md

> You can allow users to type in the DropDownBox text field to add more values to the List. In this tutorial, a press on the Enter key passes values to the List. To implement this functionality, enable the [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#acceptCustomValue) property, disable the [openOnFieldClick](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#openOnFieldClick) property, and handle the [enterKey](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onEnterKey) event. In this handler, update the shared data source and [reload](/Documentation/ApiReference/UI_Components/dxList/Methods/#reload) the List.

Reviewer: > a press on the Enter key

Too much words for such a simple action...

> The Enter keystroke

```suggestion
You can allow users to type in the DropDownBox text field to add more values to the List. In this tutorial, the Enter keystroke passes values to the List. To implement this functionality, enable the [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#acceptCustomValue) property, disable the [openOnFieldClick](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#openOnFieldClick) property, and handle the [enterKey](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onEnterKey) event. In this handler, update the shared data source and [reload](/Documentation/ApiReference/UI_Components/dxList/Methods/#reload) the List.
```
